### ISSUE TITLE
Runner Environment Variables seem to mangle Base64 Values

### ISSUE THREAD

**Issue by andrewconnell** [wallabyjs/public#742]
### Issue description or question
My app heavily leverages on environment variables for configuration (microservice based app). Some of the values I have to pass in are base64 strings. Setting these values from the command line (without wallaby) works just fine, both in running the app & in running the tests.

However, when running it in Wallaby, it seems that the string is getting malformed... suspect due to the presence of equal signs in the Base64 string. One of the strings I pass in always has two `=` on the end... but when the tests get the string from the environment variable, they two trailing `=` are missing... yet when I run it outside of wallaby, it breaks.

Could it be that internally you're splitting the env vars up from name=value on the equals sign? When I replace those two characters with underscores, my test can see those two characters just fine.

### Wallaby.js configuration file
*Including only the part for the runner...*

```javascript
        this.env = {
            type: 'node',
            params: {
                env: 'ACCOUNT_KEY=yeRWCr2p6r5Y+IDWsOqhFZIv7y54Nae1rr8CtWPJRpCOQmS5GTR4jjdOolNsTSQPAiS03usyXl0ZqXRmvUPWYA==;ACCOUNT_ID=someValue'
            }
        };

```

### Code editor or IDE name and version

Visual Studio Code v1.4.0

### OS name and version

OSX v10.11.6

**Comment by ArtemGovorov** [comment]
> Could it be that internally you're splitting the env vars up from name=value on the equals sign

It does sound like a bug.

There's an easy workaround for node env variables:
```javascript
this.env = {
            type: 'node'
        };
process.env.ACCOUNT_KEY = 'yeRWCr2p6r5Y+IDWsOqhFZIv7y54Nae1rr8CtWPJRpCOQmS5GTR4jjdOolNsTSQPAiS03usyXl0ZqXRmvUPWYA==';
process.env.ACCOUNT_ID='someValue';
```
Wallaby will just pass the current process env variables to test workers.

**Comment by andrewconnell** [comment]
Awesome... the workaround works for me for now... thanks.

**Comment by ArtemGovorov** [comment]
The issue is fixed in core v1.0.279.
